python/tvm/relay/frontend/tflite.py

```diff
@@ -153,13 +155,24 @@ def get_tensors(self, tensors_idx_list):
         return_list = list()
         for tensor_idx in tensors_idx_list:
             if tensor_idx < 0:
-                return_list.append(TensorWrapper(tensor_idx, 0, 0))
+                return_list.append(TensorWrapper(tensor_idx, 0, 0, None))
                 continue
 
             tensor = self.subgraph.Tensors(tensor_idx)
             buffer_idx = tensor.Buffer()
             buffer = self.model.Buffers(buffer_idx)
-            return_list.append(TensorWrapper(tensor_idx, tensor, buffer))
+
+            # Check if the tensors are quantized. Parse if yes.
+            qnn_params = None
+            tflite_qnn_params = tensor.Quantization()
+            if tflite_qnn_params is not None:
+                qnn_params = dict()
+                qnn_params['scale'] = float(tflite_qnn_params.ScaleAsNumpy())
+                qnn_params['zero_point'] = int(tflite_qnn_params.ZeroPointAsNumpy())
+                # Check that the scale and zero points are valid.
+                if qnn_params['scale'] == 0 and qnn_params['zero_point'] == 0:
+                    qnn_params = None
+            return_list.append(TensorWrapper(tensor_idx, tensor, buffer, qnn_params))
```

Comment:
Better way is to check `if qnn_params['scale'] != 0 or qnn_params['zero_point'] != 0:`, because the default value of `qnn_params` is `None` in line 166.